src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,16 @@
+package racingcar;
+
+public class Car {
+    private final int moveCount = 4;
```

Comment:
상수는 대문자로 선언되어야 합니다 😄 
```suggestion
    private final int MOVE_COUNT = 4;
```

Reply:
네이밍 컨벤션을 보고 수정했습니다.!